### ISSUE TITLE
"Last Updated entry" gives goaccess startup time, not time of last update.

### ISSUE THREAD

**Issue by SleepyMario** [allinurl/goaccess#1356]
Hello. The "last updated" entry on top of the html life graph is displaying the time of startup of goaccess instead of the 'last updated' time. This makes sense because I use tail to parse the most current log file, but do you know of a way to make it actually display the real 'last updated' time when it's doing a live update? You can see the live-html feed at https://sleepymario.com/all/site-statistics/ if you want to see it.
This is goaccess v1.3 on Gentoo. The command I use is below:

zcat /PATH/TO/log.gz | goaccess /PATH/TO/log \
--geoip-database=/PATH/TO/GeoLite2-City.mmdb \
--ignore-panel REQUESTS_STATIC \
--ignore-panel REQUESTS \
--ignore-panel NOT_FOUND \
--ignore-panel STATUS_CODES \
--ignore-panel KEYPHRASES \
--ignore-referer=blah.com \
--ignore-referer=blah2.com \
--ignore-referer=blah3.com \
--ignore-referer=blah4.com \
--ignore-referer=blah5.com \
--log-format=COMBINED \
--ignore-crawlers \
--anonymize-ip \
--exclude-ip 192.168.0.1-192.168.0.255 \
--exclude-ip 192.168.1.1-192.168.1.255 \
--exclude-ip 192.168.2.1-192.168.2.255 \
--daemonize \
--double-decode \
--real-os \
--ssl-cert=/PATH/TO/CERT \
--ssl-key=/PATH/TO/KEY \
--pid-file=/PATH/TO/goaccess.pid \
-a -o /PATH/TO/report.html --real-time-html -

**Comment by allinurl** [comment]
Yep, I can confirm this. I'll look into it and post back asap. Thanks for reporting it!
